config/cl_config.lua

```diff
@@ -34,39 +34,45 @@ Config.zoneCheck = 1 -- Seconds to check if in the zone.
-[18] = {name = "Downtown Vinewood", x = 638.73327636719, y = 206.56455993652, z = 97.604179382324},
-[19] = {name = "Pillbox Hill", x = 65.847717285156, y = -616.36883544922, z = 30.931786804199}
+	[1] = {name = "Legion", pos = vector3(217.58, -802.31, 30.76)},
+	{name = "Pink Cage", pos = vector3(273.77, -344.17, 44.91)},
+	{name = "Mirror Park", pos = vector3(1036.25, -763.19, 57.99)},
+	{name = "Pillbox Hill", pos = vector3(44.58, -843.09, 31.127)},
+	{name = "Mission Row", pos = vector3(458.22, -895.58, 35.97)},
+	{name = "Alta Street", pos = vector3(-280.97, -887.98, 31.31)},
+	{name = "Mission Row", pos = vector3(453.33, -1146.18, 29.51)},
+	{name = "East Vinewood", pos = vector3(984.27, -206.19, 71.06)},
+	{name = "Underground Garage", pos = vector3(126.01, -714.44, 33.13)},
+	{name = "Davis", pos = vector3(-54.26, -1835.74, 26.57)},
+	{name = "Banning", pos = vector3(-36.02, -2096.27, 16.93)},
+	{name = "Davis", pos = vector3(-73.31, -2004.03, 18.27)},
+	{name = "Vespuci Beach", pos = vector3(-1183.96, -1509.88, 4.64)},
+	{name = "Del Perro", pos = vector3(-2030.96, -465.92, 11.60)},
+	{name = "West Vinewood", pos = vector3(-554.99, 331.64, 84.37)},
+	{name = "West Vinewood", pos = vector3(-340.53, 266.64, 85.67)},
+	{name = "Downtown Vinewood", pos = vector3(362.45, 298.58, 103.88)},
+	{name = "Downtown Vinewood", pos = vector3(638.73, 206.56, 97.60)},
+	{name = "Pillbox Hill", pos = vector3(65.84, -616.36, 30.93)}
 }
 
 Config.spawnLocations = { -- The index number corresponds to the garage number.
-[1] = {name = "Legion", x = 227.46684265137, y = -811.34936523438, z = 29.527854232788, heading = 161.44157409668},
-[2] = {name = "Pink Cage", x = 286.7854309082, y = -345.6882019043, z = 43.949834136963, heading = 154.09780883789}
+	[1] = {name = "Legion", x = 227.46684265137, y = -811.34936523438, z = 29.527854232788, heading = 161.44157409668},
+	[2] = {name = "Pink Cage", x = 286.7854309082, y = -345.6882019043, z = 43.949834136963, heading = 154.09780883789}
 }
 
-Config.barierLocations = {
-[1] = {model = "prop_sec_barier_02b", x = 266.09, y = -349.35, z = 44.74}, -- Pink Cage 1
-[2] = {model = "prop_sec_barier_02a", x = 285.28, y = -355.78, z = 45.13}, -- Pink Cage 2
-[3] = {model = "prop_sec_barrier_ld_01a", x = 206.52, y = -803.47, z = 30.95}, -- Legion 1
-[4] = {model = "prop_sec_barrier_ld_01a",x = 230.96, y = -816.36, z = 30.32}, -- Legion 2
-[5] = {model = "prop_sec_barrier_ld_01a",x = 223.87022399902, y = -740.16491699219, z = 33.428574829102}, -- Legion 3
-[6] = {model = "prop_sec_barier_02b", x = 1038.61, y = -763.18, z = 57.77}, -- Mirror Park 1
-[7] = {model = "prop_sec_barier_02b", x = 1030.61, y = -758.09, z = 57.77}, -- Mirror Park 2 
-[8] = {model = "prop_sec_barier_02b", x = 451.45919799805, y = -1144.5382080078, z = 29.356086044312}, -- Mission Row
+Config.barrierLocations = {
+	[1] = {pos = vector3(266.09, -349.35, 44.74)}, -- Pink Cage 1
```

Comment:
```suggestion
	{pos = vector3(266.09, -349.35, 44.74)}, -- Pink Cage 1
```
Might as well fix it here too 🤔